fiftyone/server/db/_mongo_lightning.py

```
@@ -0,0 +1,627 @@
+    match = None
+    matcher = lambda v: False
+    if query.search:
+        match = query.search
+        matcher = lambda v: not v.startswith(match)
+        if query.is_object_id_field:
+            match = match[:_TWENTY_FOUR]
+            matcher = lambda v: v < match
+
+    try:
+        result = await collection.distinct(query.path)
+    except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)